functests/test_resources/door_entry_story.yaml

```diff
@@ -129,21 +136,22 @@ steps:
   - step:
       action: ASSETS_CREATE_IF_NOT_EXISTS
       description: Create a front door in Paris at Gare du Nord Apartments
-    signature:
-      attributes:
-        arc_display_name: Gare du Nord Apartments Paris Front Door
+    selector:
+      - attributes.arc_display_name
     behaviours:
       - RecordEvidence
       - Attachments
     attributes:
+      arc_display_name: Gare du Nord Apartments Paris Front Door 2022-03-04
       arc_display_type: door
       arc_firmware_version: "1.0"
       arc_serial_number: das-x4-04
       arc_description: Electronic door entry system controlling the front residential entrance to Apartements du Gare du Nord
       wavestone_asset_id: front.gdn.paris.wavestonedas
     location:
-      signature:
-        display_name: Apartements du Gare du Nord
+      selector:
+        - display_name
+      display_name: Apartements du Gare du Nord 2022-03-04
```

Comment:
arc_ again. I'm going to stop pointing these out but there are more

Reply:
they are all the display name of 'location' which does not have arc_ prefix.....